documentation/translations/coordinating.rst

```diff
@@ -3,79 +3,201 @@ Coordinating
+will be found at ``docs.python.org/LANG/``, but not in the switcher.
+
+This will be done XXX
+
+.. XXX When ...? Discussion needed...
+.. My idea: Time based, e.g. 2 months of activity, showing that they aren't going anywhere
+
+Adding to the language switcher
+-------------------------------
+
+.. XXX https://github.com/python/devguide/issues/1586
+
+Once the following resources have been fully translated on the XXX branch:
+
+- ``bugs.po``, with proper links to the language repository issue tracker
+- all files in the ``tutorial/`` folder
+- ``library/functions.po``, the page documenting builtins
+
+The translation can be added to the language switcher. This can be done with a
+pull request to docsbuild-scripts_, like `this commit <https://github.com/python/docsbuild-scripts/commit/e4a8aff9772738a63d0945042777d18c3d926930>`__
+if your translation was previously built but not in the switcher, or like
+`this commit <https://github.com/python/docsbuild-scripts/commit/a601ce67c6c2f3be7fde3376d3e5d3851f19950b>`__
+if this is it's initial addition.
 
 
 PEP 545 summary
 ===============
 
 Here are the essential points of :PEP:`545`:
 
-- Each translation is assigned an appropriate lowercased language tag,
-  with an optional region subtag, and joined with a dash, like
-  ``pt-br`` or ``fr``.
+- Each translation is assigned an appropriate lowercase
+  `IETF language tag <https://datatracker.ietf.org/doc/html/rfc5646.html>`_.
+  The tag may have an optional region subtag, joined with a dash.
+  For example, ``pt`` (Portuguese) or ``pt-br`` (Brazilian Portuguese).
+
+- Each translation is under CC0 and is marked as such in the README.
+
+- Translation files are hosted in repositories under the Python org:
+  ``https://github.com/python/python-docs-{LANGUAGE_TAG}``
 
-- Each translation is under CC0 and marked as such in the README (as in
-  the cookiecutter).
+- Translations having completed ``bugs``, ``tutorial/``
+  and ``library/functions`` are added to the language switcher.
 
-- Translation files are hosted on
-  ``https://github.com/python/python-docs-{LANGUAGE_TAG}`` (not
-  mandatory to start a translation, but mandatory to land on
-  ``docs.python.org``).
 
-- Translations having completed ``tutorial/``, ``library/stdtypes``
-  and ``library/functions`` are hosted on
-  ``https://docs.python.org/{LANGUAGE_TAG}/{VERSION_TAG}/``.
+Translating Sphinx
+==================
+
+Some messages that appear in the docs can not be translated, this is because they
+are either part of the theme, which currently cannot be translated (see this
+`issue <https://github.com/python/python-docs-theme/issues/194>`__),
+they are part of Sphinx, which requires them to be translated in the
+`sphinx-doc Transifex <https://app.transifex.com/sphinx-doc/>`__.
+Coordinators should direct some translators there, so that the documentation
+is fully translated.
+
+.. Where should this go? Coordinators or Translators
+.. Should we share coordinators?
 
 
 Transifex
 =========
```

Comment:
There's already a section about Transifex above, maybe the two could be merged or this could become a subsection/aside?

Reply:
I would also merge those.